src/pages/en/guides/markdown-content.md

```diff
@@ -238,32 +240,65 @@ You can import Markdown and MDX files directly into your Astro files. This gives
 
 You can import one specific page with an `import` statement, or multiple pages with [`Astro.glob()`](/en/guides/imports/#astroglob).
 
-```mdx title="/src/pages/posts/great-post.mdx"
+```astro title="src/pages/index.astro" {3,6}
```

Comment:
I think this is a case where we’re highlighting everything, so maybe no highlighting is better?

````suggestion
```astro title="src/pages/index.astro"
````